test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -277,6 +277,21 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 				);
 			} );
 
+			step( 'Can see the Jetpack blocks', async function() {
+				// Jetpack blocks are broken in IE11. See https://github.com/Automattic/jetpack/issues/14273
+				if ( dataHelper.getTargetType() === 'IE11' ) {
+					return this.skip();
+				}
+				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
+				await gEditorComponent.openBlockInserterAndSearch( 'Jetpack' );
+				assert.strictEqual(
+					await gEditorComponent.isBlockCategoryPresent( 'Jetpack' ),
+					false,
```

Comment:
Just noted this should be `true`. Changing in it a bit, but also checking why the test is passing.

Reply:
Fixed in 52fbf2d